src/main/java/frc/robot/commands/ScoreNoteCommand.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ScoringConstants;
+import frc.robot.subsystems.Scoring;
+
+public class ScoreNoteCommand extends Command {
+    private Scoring scoring = Scoring.getInstance();
+    private Timer timer = new Timer();
+
+    public ScoreNoteCommand() {
+        this.addRequirements(scoring);
+    }
+
+    @Override
+    public void initialize() {
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        scoring.runRoller(0);
+    }
+
+    @Override
+    public boolean isFinished() {
+        if (timer.get() > ScoringConstants.kSecondsToScore) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        scoring.stopRoller();
+        timer.stop();
```

Comment:
Add a timer.reset here, otherwise the timer will continue at the stopped time the second time the command runs 

Double check the description of timer.reset because I think reset also stops the timer in which case you don't need the stop